core/tests/performance_framework/perf_services.py

```diff
@@ -240,8 +240,11 @@ def _setup_driver(self, proxy=None, use_proxy=False):
             chrome_options = webdriver.ChromeOptions()
             # Disable several subsystems which run network requests in the
             # background. This helps reduce noise when measuring network
-            # performance.
+            # performance. Also, disable prerendering by chrome which renders
+            # a page in the background leading to wrong test results.
```

Comment:
Yes, with prerender we send more bits and that led to increase in the total size when testing a page. Hence, we disable prerender for preformance tests.